gcli/src/cmd/send.rs

```diff
@@ -56,6 +56,7 @@ pub struct Send {
 impl Send {
     pub async fn exec(&self, signer: Signer) -> Result<()> {
         signer
+            .calls
```

Comment:
why send_message is from "calls", but `transfer` is from `.balance`?

Reply:
I think all the calls intended to communicate with programs should be placed in `calls` and all the calls working with balance in `balance`

Reply:
Isn't it better to split them by pallet names?

Reply:
Hmm, But will it make sense for API user that's not familiar with gear pallet organisation? There will be 7 fields in `Signer`, 1 per pallet, + `utils` + `rpc`